sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,21 @@
+--
+DELETE FROM `gossip_menu` WHERE `MenuID` IN (7583,7585,7586);
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`, `VerifiedBuild`) VALUES
+(7583,9227,0),(7585,9229,0),(7586,9231,0);
```

Comment:
I think you misunderstood it.
Remove this section completely.

Reply:
ah yeah that makes sense